### PR TITLE
Update technical-preview-free-tier.md

### PR DESCRIPTION
**Pull request by danieltprice**
- Add information about compute hour limit and data access.
- Glossary updates to support links from Free Tier content
https://neon-next-git-dprice-elaborate-on-compute-h-73a444-neondatabase.vercel.app/docs/introduction/technical-preview-free-tier#user-content-fn-1

### CODE REVIEW

**Review by acervantes23 — DISMISSED** [review]

File: content/docs/reference/glossary.md [hunk 1]
```diff
@@ -36,13 +36,17 @@ A periodic load generated by the Control Plane to determine if a compute can sta
 
 A [copy-on-write](#copy-on-write) clone of a Neon project's primary branch or previously created child branch. A branch can be created from the current or past state of the parent branch. A branch created from the current state of the parent branch includes the databases and users that existed in the parent branch at the time of branch creation. A branch created from a past state of the parent branch includes the databases and users that existed in the past state. The data in a branch can be modified independently from its originating data. See [Branching](/docs/introduction/branching). Each branch has a dedicated [compute endpoint](#compute-endpoint), which is the compute instance associated with the branch. Connecting to a database in a branch requires connecting via the branch's compute endpoint. For more information, see [Connect to a branch](/docs/manage/branches#connect-to-a-branch).
 
+### Child branch
+
+A [branch](#branch) created from a [primary branch](#primary-branch) or another branch in your Neon project.
```
Comment (stepashka) [inline content/docs/reference/glossary.md:41]: a child branch can have it's own child
this definition doesn't work well for the children of direct descendants of primary branch

File: content/docs/introduction/technical-preview-free-tier.md [hunk 2]
```diff
@@ -10,13 +10,11 @@ The Neon cloud service is available for free during the Technical Preview. For i
 The Technical Preview Free Tier has the following limits:
 
 - 1 Neon project
-- 10 branches, with a limit of 3GB of data per branch
-- A compute endpoint for each branch, each with up to 1 vCPU and 4GB of RAM (with 512MB of RAM allocated to PostgreSQL shared buffers)
-- 100 compute hours per month (total)
+- 10 branches including the [primary branch](/docs/reference/glossary#primary-branch), with a limit of 3GB of data per branch
+- A [compute endpoint](/docs/reference/glossary#compute-endpoint) for each branch, each with up to 1 vCPU and 4GB of RAM (with 512MB of RAM allocated to PostgreSQL shared buffers)
+- 100 compute hours per month (total). Neon scales compute resources to zero after 5 minutes of inactivity. Only active compute hours are counted toward the compute hour limit. When you reach the limit, compute endpoints assigned to [child branches](/docs/reference/glossary#child-branch) are unavailable until the limit is reset at the beginning of the next month. You are always able to connect to the compute endpoint assigned to the primary branch, regardless of the compute hour limit. This ensures that access to data on the primary branch is never interrupted.
```
Comment (stepashka) [inline content/docs/introduction/technical-preview-free-tier.md:15]:
```suggestion
- 100 compute hours per month (total). Neon scales compute resources to zero after 5 minutes of inactivity. Only active compute hours are counted toward the compute hour limit. When you reach the limit, compute endpoints assigned to [non-primary branches](/docs/reference/glossary#child-branch) are unavailable until the limit is reset at the beginning of the next month. You are always able to connect to the compute endpoint assigned to the primary branch, regardless of the compute hour limit. This ensures that access to data on the primary branch is never interrupted.
```

File: content/docs/reference/glossary.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/reference/glossary.md]: Changed definition to "non-primary branch"

File: content/docs/introduction/technical-preview-free-tier.md [hunk 3]
```diff
@@ -10,13 +10,11 @@ The Neon cloud service is available for free during the Technical Preview. For i
 The Technical Preview Free Tier has the following limits:
 
 - 1 Neon project
-- 10 branches, with a limit of 3GB of data per branch
-- A compute endpoint for each branch, each with up to 1 vCPU and 4GB of RAM (with 512MB of RAM allocated to PostgreSQL shared buffers)
-- 100 compute hours per month (total)
+- 10 branches including the [primary branch](/docs/reference/glossary#primary-branch), with a limit of 3GB of data per branch
+- A [compute endpoint](/docs/reference/glossary#compute-endpoint) for each branch, each with up to 1 vCPU and 4GB of RAM (with 512MB of RAM allocated to PostgreSQL shared buffers)
+- 100 compute hours per month (total). Neon scales compute resources to zero after 5 minutes of inactivity. Only active compute hours are counted toward the compute hour limit. When you reach the limit, compute endpoints assigned to [non-primary branches](/docs/reference/glossary#non-primary-branch) are unavailable until the limit is reset at the beginning of the next month. You are always able to connect to the compute endpoint assigned to the primary branch, regardless of the compute hour limit. This ensures that access to data on the primary branch is never interrupted.
```
Comment (stepashka) [inline content/docs/introduction/technical-preview-free-tier.md:15]:
```suggestion
- 100 hours of Free Tier compute per month (total). Neon scales Free Tier compute to zero after 5 minutes of inactivity. Only active time of Free Tier compute is counted toward the limit. When you reach the limit, Free Tier compute endpoints assigned to [non-primary branches](/docs/reference/glossary#non-primary-branch) are unavailable until the limit is reset at the beginning of the next month. You are always able to connect to the compute endpoint assigned to the primary branch, regardless of the limit. This ensures that access to data on the primary branch is never interrupted.
```

**Review by acervantes23 — DISMISSED** [review]

**Review by stepashka — APPROVED** [review]

**Pull request merged**